server/infra/database/GrowerAccountRepository.ts

```diff
@@ -33,6 +33,20 @@ export default class GrowerAccountRepository extends BaseRepository<GrowerAccoun
       result.whereIn(whereIn.field, whereIn.values);
     });
 
+    Object.entries(parameters).forEach(([key, val]) => {
+      if (val === 'not null') {
```

Comment:
Where is this been set to 'null' or 'not null' ? Are we expecting it from the frontend? If we are, would this not error out in the case of uuids, since Joi has that validation check?